indexer/ftypes_matcher.hpp

```diff
@@ -173,6 +173,16 @@ class IsHotelChecker : public BaseChecker
   static vector<string> const & GetHotelTags();
 };
 
+// WiFi is a type in classificator.txt, so it presence in a
+// FeatureType and should be checked for MapObject filling.
```

Comment:
"so it presence in a FeatureType and should be checked", sorry? Could you please paraphrase the comment, I can't get to it's meaning.